lib/views/pay/pay_view.dart

```diff
@@ -169,10 +182,13 @@ class _PayViewState extends State<PayView> {
               width: 200,
               onPress: () async {
                 ClipboardData? data = await Clipboard.getData('text/plain');
+                if (data == null) {
+                  return;
```

Comment:
when this is null?

Reply:
It is when we don't have anything copied to the clipboard. Handles the null check exception.

Reply:
Nice, so we can show a Snackbar and post a small message "nothin' to in the clipboard"

Return in this way it is not good